rudder-webapp/SPECS/rudder-webapp.spec

```diff
@@ -381,16 +379,6 @@ echo "INFO: Launching script to check if a migration is needed"
 %{rudderdir}/bin/rudder-upgrade
 echo "INFO: End of migration script"
 
-# Create and populate technique store
-if [ ! -d /var/rudder/configuration-repository ]; then mkdir -p /var/rudder/configuration-repository; fi
-if [ ! -d /var/rudder/configuration-repository/shared-files ]; then mkdir -p /var/rudder/configuration-repository/shared-files; fi
-if [ ! -d /var/rudder/configuration-repository/techniques ]; then
-	cp -a %{rudderdir}/share/techniques /var/rudder/configuration-repository/
```

Comment:
Not sure about these lines, but I don't see in which conditions they can be useful.